### PR TITLE
really just a lot of shit about bows

### PR DESCRIPTION
**Pull request by Hatterhat**
## About The Pull Request
- makes the pipe bow draw sprite not have two bowstrings
- the ash walker nest now starts with 3 slabs, in case the ash walkers completely forgot about that
- holy shit how much grammar did i fuck with
## Why It's Good For The Game
sprite bad. bows neat, recipe jank, hamfisted attempt to increase proliferation.
## Changelog
:cl:
add: The Ash Walkers' nest on Lavaland now starts with three bowyery slabs.
add: You can now welder-harden arrows. It might take longer.
balance: Silkstring's costs adjusted for bows and whatnot.
spellcheck: Grammar adjusted on a lot of things relating to bows.
fix: Pipe bows' bowstring doesn't look like it replicated itself upon draw.
/:cl:

### CODE REVIEW

**Comment by Hatterhat** [comment]
how do i tell the bot to update the tags. its fucked.

File: _maps/map_files/Mining/Lavaland.dmm [hunk 1]
```diff
@@ -2034,6 +2034,13 @@
 /obj/structure/stone_tile,
 /turf/open/indestructible/boss,
 /area/ruin/unpowered/ash_walkers)
+"DQ" = (
+/obj/structure/stone_tile{
+	dir = 4
+	},
+/obj/item/book/granter/crafting_recipe/bone_bow,
```
Comment (Trilbyspaceclone) [inline _maps/map_files/Mining/Lavaland.dmm:2041]: This one is rather bad

File: _maps/map_files/Mining/Lavaland.dmm [hunk 2]
```diff
@@ -2401,6 +2408,7 @@
 	},
 /obj/structure/table/wood,
 /obj/item/kitchen/knife/combat/bone,
+/obj/item/book/granter/crafting_recipe/bone_bow,
```
Comment (Trilbyspaceclone) [inline _maps/map_files/Mining/Lavaland.dmm:2411]: Why are you just giving out the crafting books?

File: _maps/map_files/Mining/Lavaland.dmm [hunk 3]
```diff
@@ -2507,6 +2515,11 @@
 /obj/structure/stone_tile/surrounding_tile/cracked,
 /turf/closed/mineral/volcanic/lava_land_surface,
 /area/ruin/unpowered/ash_walkers)
+"WA" = (
+/obj/structure/stone_tile/block/cracked,
+/obj/item/book/granter/crafting_recipe/bone_bow,
```
Comment (Trilbyspaceclone) [inline _maps/map_files/Mining/Lavaland.dmm:2520]: You only ever need one, and your ment to have to trade for one not have to steal it.

File: code/datums/components/crafting/recipes/recipes_weapon_and_ammo.dm [hunk 4]
```diff
@@ -291,17 +291,17 @@
 /datum/crafting_recipe/arrow
 	name = "Arrow"
 	result = /obj/item/ammo_casing/caseless/arrow
-	time = 40
```
Comment (Trilbyspaceclone) [inline code/datums/components/crafting/recipes/recipes_weapon_and_ammo.dm:294]: They take time for a reason.

File: code/datums/components/crafting/recipes/recipes_weapon_and_ammo.dm [hunk 5]
```diff
@@ -291,17 +291,17 @@
...
+	time = 20
 	reqs = list(/obj/item/stack/sheet/mineral/wood = 1,
-				 /obj/item/weaponcrafting/silkstring = 1,
```
Comment (Trilbyspaceclone) [inline code/datums/components/crafting/recipes/recipes_weapon_and_ammo.dm:296]: They should need to tie the arrow head with something like a strong piece of woven cloth, not just any piece.

**Review by Trilbyspaceclone — APPROVED** [review]
I really like the changes with words and stuff!

**Review by silicons — APPROVED** [review]

**Pull request merged**